components/profile-controller/controllers/profile_controller.go

```diff
@@ -369,6 +404,155 @@ func (r *ProfileReconciler) getAuthorizationPolicy(profileIns *profilev1.Profile
 	}
 }
 
+
+// updateAnyUIDScc Update anyuid SCC for Openshift installation
+// this allows any pod running in new profile namespaces to run as anyuid for user and group
+// Ths is needed for both Istio side car and Knative in OCP 4.6/4.7
+func (r *ProfileReconciler) updateAnyUIDScc(ctx context.Context, profileIns *profilev1.Profile) error {
+	logger := r.Log.WithValues("profile", profileIns.Name)
+	logger.Info("Creating anyuid SCC for profile")
+	//ctx := context.Background()
+	var priority int32 = 10
+	//actual, err := client.SecurityContextConstraints().Create(ctx, required, metav1.CreateOptions{})
+	scc := &securityv1.SecurityContextConstraints{
```

Comment:
`ApiVersion` and Kind should be defined as part of SCC resource. The new resource would be :
```
scc := &securityv1.SecurityContextConstraints{

		TypeMeta: metav1.TypeMeta{
			APIVersion: "security.openshift.io/v1",
			Kind: "SecurityContextConstraints",
		},
		ObjectMeta: metav1.ObjectMeta{
			Name: "scc-"+profileIns.Name,
		},
		Priority:                 &priority,
		AllowPrivilegedContainer: true,
		AllowHostNetwork:         false,
		AllowHostPorts:           false,
		//Volumes:                  ["*"],
		SELinuxContext: securityv1.SELinuxContextStrategyOptions{
			Type: securityv1.SELinuxStrategyMustRunAs,
		},
		RunAsUser: securityv1.RunAsUserStrategyOptions{
			Type: securityv1.RunAsUserStrategyRunAsAny,
		},
		FSGroup: securityv1.FSGroupStrategyOptions{
			Type: securityv1.FSGroupStrategyRunAsAny,
		},
		//This needs to be removed, since sa default-editor is attached to notebooks and it fails to use volume when an scc is present
		//Users: []string{"system:serviceaccount:"+profileIns.Name, "system:serviceaccount:"+profileIns.Name+":default-editor"},
		// switched to default that is used by kfserving for serving pods
		Users: []string{"system:serviceaccount:"+profileIns.Name, "system:serviceaccount:"+profileIns.Name+":default"},
		Groups: []string{"system:cluster-admins"},
	}
```